samples/cellular/modem_shell/src/location/location_srv_ext_nrf_cloud.c

```diff
@@ -113,8 +118,11 @@ static int location_srv_ext_nrf_cloud_location_get(
 	}
 
 	mosh_print("Sending positioning request to cloud via MQTT");
+	/* Below, no need to send conf if it matches the cloud defaults. */
 	err = nrf_cloud_location_request(
-		cell_data, scanning_results, (callback != NULL), callback);
+		cell_data, scanning_results,
+		(!conf.do_reply || !conf.fallback || conf.hi_conf) ? &conf : NULL,
```

Comment:
The cloud team works hard to avoid changing defaults because they know older devices will be unaware of the change and may no longer work correctly.  So it is fine to make assumptions here about the cloud defaults.

It does cost something, albeit very little, to pass the additional parameters. But since this is mosh, I'll remove this optimization.

Reply:
Yes, passing parameters does cost something. But the check whether to pass the parameters or not probably costs more, unless it can be optimized out by the compiler.

Reply:
I was speaking of data costs and radio power costs accumulated over time, not CPU clock cycle costs.